data/projects/implement_aspects.md

```diff
@@ -1,29 +1,63 @@
+Initial configuration of coala is **hard**, especially for new user. There
```

Comment:
maybe make it "unnecessary hard" or "could be improved"
It's quite easy compared to setting up all the different tools yourself.